src/System/Relay/middleware/__tests__/cacheHeaderMiddleware.jest.ts

```diff
@@ -163,6 +163,33 @@ describe("cacheHeaderMiddleware", () => {
         expect(next).toHaveBeenCalledWith(req)
         expect(res).toEqual({ status: 200 })
       })
+
+      it("skips CDN cache if nocache query param present", async () => {
+        req.operation = {
+          text: `
+          query ArtistQuery @cacheable {
+            artist(id: "andy-warhol) {
+              name
+            }
+          }
+        `,
+        }
+
+        jest.spyOn(window as any, "location", "get").mockImplementation(() => ({
+          pathname: "/artists?nocache=1",
```

Comment:
Cool, so this just checks for existence. could be `nocache=true` too

Reply:
Yep, anything truthy. `?nocache=` correctly doesn't count (that won't skip the cache). but `?nocache=blah|true|1`, etc. - anything else.